es-ds-docs/pages/molecules/skeleton.vue

```diff
@@ -0,0 +1,164 @@
+<script setup lang="ts">
+const loading = ref(true);
+
+const asyncTimeout = async (seconds = 3) => {
+    const millisecondTimeout = seconds * 1000;
+    return new Promise((resolve) => {
+        setTimeout(resolve, millisecondTimeout);
+    });
+};
+
+const startLoading = async () => {
+    loading.value = true;
+    await asyncTimeout();
+    loading.value = false;
+};
+
+const esSkeletonProps = [
+    ['animation', 'String', 'wave', `Options are 'wave', 'fade', or 'none'`],
+    ['height', 'String', '1rem', `Manually set height`],
+    ['width', 'String', 'auto', `Manually set width`],
+    ['size', 'String', 'null', `Manually set width and height, overriding 'height' and 'width' props`],
+];
+
+const esSkeletonWrapperProps = [
+    ['loading', 'Boolean', 'false', `Determines whether to show loading template or default content`],
+];
+
+const esSkeletonImgProps = [
+    ['aspect', 'String', '16:9', `Adds a container around the skeleton image with given aspect ratio`],
+    ['noAspect', 'Boolean', 'false', `Override the default aspect ratio and height, width, or size`],
+    ['...EsSkeleton props', '...', '...', `Takes all 'esSkeleton' props shown above`],
```

Comment:
For a complete reference and since it would be easy in code, we could replace this line with `...esSkeletonProps` and actually list all the EsSkeleton props in this table as well.

I suppose it would then be nice to do a little sort with `localeCompare` on the prop name so they're in alphabetical order but that's simple enough to tack on to this expression.